bindings/rust/wasmedge-sdk/src/vm.rs

```diff
@@ -248,6 +254,28 @@ impl Vm {
         Ok(returns)
     }
 
+    pub fn run_func_async(
```

Comment:
Notice that the `wasmedge_sys::run_registered_function_async` method is an experimental API as it cannot return a Rust standard `future` as result. Therefore, this wrapper function can only serve as a temporary API. Once the async design is done, this wrapper and the existing low-level async APIs defined in `wasmedge_sys::Vm` will be replaced.

Reply:
Hmm ~ I think experimental API is ok to be merged. If your design ready later you can notify me, I will happily replace it in runwasi future.

Reply:
Sure! I'll let you know if the async design is ready for publishing.